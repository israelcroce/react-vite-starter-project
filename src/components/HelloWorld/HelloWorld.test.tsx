import { render } from "@testing-library/react";

import HelloWorld from "./HelloWorld";

describe("HelloWorld Snapshots Suite", () => {
  it("Render correctly", () => {
    const { baseElement } = render(<HelloWorld />);

    expect(baseElement).toMatchSnapshot();
  });
});
