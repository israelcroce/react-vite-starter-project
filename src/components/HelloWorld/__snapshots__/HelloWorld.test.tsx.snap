// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HelloWorld Snapshots Suite > Render correctly 1`] = `
<body>
  <div>
    <h1>
      Hello World!
    </h1>
  </div>
</body>
`;
